system_tests/run_system_test.py

```diff
@@ -22,6 +22,7 @@
 import bigtable
 import bigtable_happybase
 import datastore
+import logging_
```

Comment:
Not really:  it is how we've dealt with argument/parameter/attribute names shadowing builtins, which made it seem right for modules which would shadow stdlib modules.